install-dev/init.php

```diff
@@ -30,6 +30,12 @@
     die('You need at least PHP 5.4 to run PrestaShop. Your current PHP version is '.PHP_VERSION);
 }
 
+/* in dev mode - check if composer was executed */
+if ((!is_dir(_PS_ROOT_DIR_.DIRECTORY_SEPARATOR.'vendor') ||
```

Comment:
This constant seems to be defined later in this file, especially when we require the file `defines.inc.php`. Maybe you should use the constant `__DIR__` here.

```php
 if ((!is_dir(__DIR__.'/../vendor') || !file_exists((__DIR__.'/../vendor/autoload.php'))) {
```